.flowconfig

```diff
@@ -4,6 +4,7 @@
 .*/node_modules/react-motion*
 .*/node_modules/jss/*
 .*/node_modules/findup/*
+.*/node_modules/redux-form/lib/index.js.flow*
```

Comment:
FYI
This will "disable" the typings on everything related to redux-form, since all types from redux-form now will become "Object", and `type A = Object & {...stuff}` will match everything.. :/

Reply:
@erlingfn

`yarn add -D flow-bin@0.77.0 && base64 -d <<< "ZGlmZiAtLWdpdCBhL2FwcC9hY3Rpb25zL1VzZXJBY3Rpb25zLmpzIGIvYXBwL2FjdGlvbnMvVXNlckFjdGlvbnMuanMKaW5kZXggYWYwMDc3MDYuLjc3ZTUzNGY3IDEwMDY0NAotLS0gYS9hcHAvYWN0aW9ucy9Vc2VyQWN0aW9ucy5qcworKysgYi9hcHAvYWN0aW9ucy9Vc2VyQWN0aW9ucy5qcwpAQCAtMjU0LDcgKzI1NCw3IEBAIGV4cG9ydCBmdW5jdGlvbiBmZXRjaFVzZXIodXNlcm5hbWU6IHN0cmluZyA9ICdtZScpIHsKICAgfSk7CiB9CiAKLWV4cG9ydCBmdW5jdGlvbiByZWZyZXNoVG9rZW4odG9rZW46IEVuY29kZWRUb2tlbikgeworZXhwb3J0IGZ1bmN0aW9uIHJlZnJlc2hUb2tlbih0b2tlbjogRW5jb2RlZFRva2VuKTogVGh1bms8YW55PiB7CiAgIHJldHVybiBjYWxsQVBJKHsKICAgICB0eXBlczogVXNlci5SRUZSRVNIX1RPS0VOLAogICAgIGVuZHBvaW50OiAnLy9hdXRob3JpemF0aW9uL3Rva2VuLWF1dGgvcmVmcmVzaC8nLAo=" | git apply && yarn flow`

:angel:

Reply:
Thanks a lot for the help. Might be a pleb question, but why is it necessary to apply the change to fetchuser as a command ant not simply suggest the change?

Reply:
> but why is it necessary to apply the change to fetchuser as a command and not simply suggest the change?

Not necessary at all, but fun (and quick) :rocket: